wellregistry/registry/migrations/0003_country_lookups.py

```diff
@@ -0,0 +1,37 @@
+"""
```

Comment:
@ayan-usgs , this was automatically generated, correct? Ed, you should be using the makemigration command to create the additional migrations that will be needed.

Reply:
Yep, they were automatically generated.